pkg/graph/resolvers/plan_discussion.go

```diff
@@ -317,13 +362,87 @@ func sendPlanDiscussionTaggedSolutionEmail(
 	return nil
 }
 
+func GetEmailRecipients(
+	ctx context.Context,
+	logger *zap.Logger,
+	store *storage.Store,
+	modelPlanID uuid.UUID,
+) ([]string, error) {
+	planCollaborators, err := store.PlanCollaboratorGetByModelPlanID(logger, modelPlanID)
```

Comment:
I might suggest you look at how users are returned for dates changed. This logic here results in a number of database calls. Even though these are technically dataloaders, I believe they are being called synchronously. I think a bespoke call might serve us better here.